src/index.js

```diff
@@ -111,8 +111,11 @@ async function start() {
   let exitCode = 0;
 
   // check gitleaks version
-  let gitleaksVersion =
-    process.env.GITLEAKS_VERSION || (await gitleaks.Latest(octokit));
+
+  let gitleaksVersion = process.env.GITLEAKS_VERSION || "8.9.0";
```

Comment:
Not specific to this PR, but what happens if the user sets the env variable to an invalid version string by mistake? e.g. "8.90" instead of "8.9.0"

Can we catch that on line 120?